enyojs/ares-project
# ENYO-2968: fix a bug related to the permissions activation when all perm...

...issions are disabled from the UI.

If all the permissions are disabled, the tag <preference name="permissions" value="none" /> must be included in the config.xml file.

How to test : 
1 - Run a build from the master branch with all permissions disabled (from the panel Phonegap build).
2 - Install the application on an  Android device 
3 - When the installation start, Android display the permissions needed to be enabled by the application. All the android permission such as GPS, camera, ... are demanded in this case (here is the bug).
4 - Run a build from the branch "ENYO-2968" with all permissions disabled (from the panel Phonegap build).
5 - Install the application on an Android device.
6 - When the installation start, Only one permissions should be demanded, the "Internet access" permission which is needed to have the Phonegap application working.

Ready for review.
Tested on Chrome (Windows 7)
Enyo-DCO-1.1-Signed-off-by: Rafik Adiche mahmoud-rafik.adiche@hp.com.
